### PR TITLE
Feat/hng 68 create payment receipt template

### LINKED ISSUE #68: [Feature]: EMAIL TEMPLATES -> Payment Receipt Template
### Description

_Create a Payment Receipt email template according to the design using React Email_

### Acceptance Criteria

- The email template must be responsive
- The email template must be dynamic
- The email template must be done according to the design
- The email tempate must be created using React Email

### Requirements	

The following props should be available for the component

1. Title
2. Image
3. Order Summary Details
4. Payment Details
5. Any other required props

### Tests

- [x] Ensure template is responsive
- [x] Ensure template can receive dynamic data according to what will be displayed
- [x] Ensure template is displayed properly according to design and requirements

###  Tasks

- [x] Create one dynamic email template using React Email
- [x] Add props according to the type of text and data it will display (check the design to confirm what will be displayed)
- [x] Template must be responsive on Bigger screens, tablet screens and mobile screens
- [x] Create a unit test to ensure template works as expected

### TimeFrame

**The deadline for submission is Saturday 11:59PM**

### Links

[`FIGMA LINK`](https://www.figma.com/design/VEItfX6St5NSAqqNHImcxD/HNG-Boilerplate-Designs?node-id=1809-11333&m=dev).
[`REACT EMAIL DOCS`](https://react.email/docs/components/html).

# IMAGES
Payment receipt mobile (1)
Payment receipt (1)
Payment receipt mobile
Payment receipt

### PR DESCRIPTION
**Pull request by fee-sah-yor**
# Description

**Closes #68**

# Changes proposed

## What were you told to do?
Create payment invoice template
Create dynamic email template using React Email
Implement responsive template on bigger screens, tablet screens and mobile screens
Ensure template is displayed properly according to design and requirements

## What did you do?

Created dynamic email template using React Email
Implemented responsive template on bigger screens, tablet screens and mobile screens
Ensured template is displayed properly according to design and requirements.
The pictures attached don't reflect the font because, as at the time of making this request, the Inter font had not been activated in codebase

**Files Created**
Email folder containing Email.tsx and EmailSecond.tsx files
Static folder containing images file

**Dependencies installed**
React Email

# Check List (Check all the applicable boxes)

🚨Please review the [contribution guideline](CONTRIBUTING.md) for this repository.

- [x] My code follows the code style of this project.
- [x] This PR does not contain plagiarized content.
- [x] The title and description of the PR is clear and explains the approach.
- [x] I am making a pull request against the **dev branch** (left side).
- [x] My commit messages styles matches our requested structure.
- [x] My code additions will fail neither code linting checks nor unit test.
- [x] I am only making changes to files I was requested to.

# Screenshots/Videos
firstlaptop_firstpart
firstlaptop_secondpart
firstmobile_firstpart
firstmobile_secondpart
firstmobile_thirdpart
laptop_firstpart
laptop_secondpart
mobile_firstpart
mobile_secondpart

### CODE REVIEW

**Comment by chumex412** [comment]
Your implementation is different from the design

File: app/styles/global.css [hunk 1]
```diff
@@ -1,12 +1,10 @@
...
 @tailwind components;
 @tailwind utilities;
 
-body {
-  font-family: "Inter", sans-serif;
-}
+
 
 @layer base {
   :root {
```
Comment (chumex412) [inline app/styles/global.css:10]: You're not allowed to edit the global.css

You should have met one of the maintainers with a request of what you needed.

**Review by chumex412 — APPROVED** [review]

**Review by chumex412 — CHANGES_REQUESTED** [review]
Please implement the fixes ASAP

**Comment by fee-sah-yor** [comment]
> Your implementation is different from the design

Do you mean the font style?

File: app/styles/global.css (hunk 1 above)
Comment (fee-sah-yor) [reply app/styles/global.css:10]: i have made the changes

**Review by shalomtaiwo — APPROVED** [review]

**Pull request merged**